modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+:image: registry.access.redhat.com/ubi8/ubi
+
+ifdef::openshift-origin[]
+:image: fedora:31
+endif::[]
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) is enabled
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `lksctp-tools` package.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a Pod that runs an SCTP listener:
+
+.. Create a file named `sctp-server.yaml` that defines a Pod using the following YAML:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  containers:
+    - name: sctpserver
+      image: {image}
+      command: ["/bin/sh", "-c"]
+      args:
+       ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]
```

Comment:
@jboxman line 42 indentation, need update to:
args:
   ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]